plugins/poll/lib/irv.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+class DiscoursePoll::Irv
+  def self.irv_outcome(poll_id)
+    poll = Poll.find(poll_id)
```

Comment:
where is this used?

Reply:
Here:

https://github.com/merefield/discourse/blob/25950403f0a48bf3b397bf6ba5a003eab318b5bf/plugins/poll/app/serializers/poll_serializer.rb#L84

I realise this is putting a lot of processing into the round trip.

One option here if we are concerned about scalability is to cache the results somehow and show the user just the latest version, and leave updates to a job.

for a handful of users, though, the processing is taking very little time.

Reply:
No I mean the poll variable, where is this used?

Reply:
@jjaffeux, thanks, yes, it is clearly redundant, good catch 👍  https://github.com/discourse/discourse/pull/27155/commits/92dfc67efc72059efaac79656bce34541bec86c4